.github/workflows/backend_checks.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     strategy:
       matrix:
         # NOTE: These are the currently supported/tested Python Versions
-        python_version: ["3.8.14", "3.9.14", "3.10.7"]
```

Comment:
If we're not testing it in CI, then we shouldn't support it as a compatible version, and that is a decision i don't think we should make without discussion first. There are other potential ramifications here we should consider first

Reply:
On speaking to @adamsachs we'll need to keep this for now in order to continue to support the masking engine in Fidesplus.